frontend/js/wordInteractions.js

```diff
@@ -136,6 +132,30 @@ function playerTypesWord () {
   });
 }
 
+function alertTypo (typedSubmission) {
+  const allTyposDiv = document.getElementById('typo-alert');
+  const thisTypoDiv = document.createElement('div');
+  const h4 = document.createElement('h4');
+  const p = document.createElement('p');
+
+  h4.textContent = `TYPO: "${typedSubmission}" is not a valid entry!`;
+
+  let lives = 'lives';
+  if (game.typos === 2) { lives = 'life'; }
+  p.textContent = `You have ${3 - game.typos} ${lives} remaining.`;
+
+  thisTypoDiv.append(h4, p);
+  allTyposDiv.appendChild(thisTypoDiv);
+  setTimeout(removeTypoAlert, 2500);
+}
```

Comment:
Look into using the MDB alert class instead

Reply:
will do!